app/javascript/components/refuels/outputSliderValue.js

```diff
@@ -0,0 +1,15 @@
+function outputValue() {
+  const inputSlider = document.querySelector("#refuel-slider");
+  const titleSpan = document.querySelector("#title-number");
+  const outputElement = document.querySelector(".refuel-slider__value");
+  const updateValue = () => {
+    outputElement.textContent = inputSlider.value;
+    titleSpan.textContent = inputSlider.value;
+  };
+  if (inputSlider) {
+    inputSlider.addEventListener("load", updateValue);
```

Comment:
```suggestion
    inputSlider.addEventListener("load", updateValue);
```
window.addEventListener("load", updateValue);

Reply:
Using **window** instead of **inputSlider** here updates the **title** and **outputElement** values to the previous slider value on page refresh.